src/test/java/io/github/sskorol/entities/TestData.java

```diff
@@ -0,0 +1,44 @@
+package io.github.sskorol.entities;
+
+import com.creditdatamw.zerocell.annotation.Column;
+import io.github.sskorol.converters.*;
+import io.github.sskorol.data.Sheet;
+import io.github.sskorol.data.Source;
+import lombok.Getter;
+import lombok.ToString;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@ToString
+@Getter
+@Source(path = "testdata.xlsx")
+@Sheet(name = "Sheet1")
+public class TestData {
+
+    @Column(name = "TC", index=0 )
```

Comment:
Reformat the code in all the affected files via hotkey please to meet the common style guides.